README.md

```diff
@@ -209,6 +221,9 @@ const toast = document.createElement('std-toast');
 console.log(toast.open); // false
 ```
 
+The `type` enumerated attribute is reflected [limited to only known values](https://html.spec.whatwg.org/multipage/common-dom-interfaces.html#limited-to-only-known-values),
```

Comment:
In the text this shows up looking like "the `type` enumerated attribute is reflected limited to only known values", there should probably be parens around the `limited to only known values` portion.